doc/examples/viz_surface_actor.py

```diff
@@ -0,0 +1,99 @@
+"""
+==================================
+Visualization of 3D Surface Rendered ROI with Streamlines
```

Comment:
I would suggest to change the title to 
Visualization of ROI contour or surface..

Reply:
"Visualization of ROI Surface Rendered with Streamlines"